Userland/Applications/ClockSettings/TimeZoneSettingsWidget.cpp

```diff
@@ -156,13 +157,7 @@ Optional<Gfx::FloatPoint> TimeZoneSettingsWidget::compute_time_zone_location() c
     return Gfx::FloatPoint { mercadian_x, mercadian_y };
 }
 
-void TimeZoneSettingsWidget::set_time_zone() const
+void TimeZoneSettingsWidget::set_time_zone()
```

Comment:
What's this change needed for?

Reply:
`spawn_or_show_error()` has to take in a non-cost `GUI::Window*` pointer (since that's what `GUI::MessageBox::show_error()` takes). If the function is marked const `window()` only returns `GUI::Window const *`

Reply:
That's unfortunate, but ok